cluster/addons/fluentd-elasticsearch/es-image/elasticsearch_logging_discovery.go

```diff
@@ -52,7 +53,13 @@ func flattenSubsets(subsets []corev1.EndpointSubset) []string {
 	ips := []string{}
 	for _, ss := range subsets {
 		for _, addr := range ss.Addresses {
-			ips = append(ips, fmt.Sprintf(`"%s"`, addr.IP))
+			if ip := net.ParseIP(addr.IP); ip != nil {
```

Comment:
nit, you have a function in k/utils that is doing this
https://github.com/kubernetes/utils/blob/6ca3b61696b65b0e81f1a39b4937fc2d2994ed6a/net/net.go#L121

Reply:
Good! Thanks @aojea